safetensors/src/lib.rs

```diff
@@ -285,6 +301,27 @@ pub enum Dtype {
     F64,
 }
 
+impl Dtype {
+    /// Gives out the size (in bytes) of 1 element of this dtype.
+    pub fn size(&self) -> usize {
+        match self {
```

Comment:
We should probably re-expose that to Python so we don't have to write it over there  (jsut to avoid maintaining 2 lists)